docs/system-models.rst

```diff
@@ -43,6 +43,12 @@ necessary to specify a particular storage systems.
     The ``InputQuantities`` and ``OutputQuantities`` classes define
     the required names for the time, current, and voltage.
 
+.. note::
+
+    Moirae's models assume that a positive current for batteries charging,
```

Comment:
Maybe it's because I am not a native English speaker, but this sounds weird to me. For international clarity, would it make more sense to say something like "Moirae's models assume that a positive current corresponds to charging of the battery, the opposite convention used by the battery-data-toolkit [...]"

Reply:
Don't worry. That sentence was bad for native speakers. I missed a word and forgot to proofread. I'll use your version